frontend/app/(no-footer)/organiser/event/dashboard/page.tsx

```diff
@@ -132,7 +132,7 @@ export default function OrganiserDashboard() {
 
   return (
     <div className="w-screen mt-16 mb-10 ml-7 h-screen max-h-screen overflow-hidden">
-      <OrganiserNavbar currPage={""} />
+      <OrganiserNavbar currPage={"EventDashboard"} />
```

Comment:
Curr page needs to be a ENUM, we need to standardise these strings.

Reply:
:( brian griefing me fr